website/app/styles/components/color-swatch.scss

```
@@ -0,0 +1,92 @@
+.doc-color-swatch {
+  display: grid;
+  grid-template-rows: 64px 1fr;
+  width: 100%;
+  margin: 16px 0;
+  padding: 0;
+  border: 1px solid #dbdbdc;
+  border-radius: 3px;
+
+
+  @media screen and (min-width: 1280px) {
+    grid-template-rows: 1fr;
+    grid-template-columns: (minmax(80px, 80px) auto); // just 80px auto didn't work, maybe a bug?
+  }
+}
+
+.doc-color-swatch__preview {
+  border-top-left-radius: 3px;
+  border-top-right-radius: 3px;
+
+  @media screen and (min-width: 1280px) {
+    grid-area: 1 / 1 / 2 / 2;
+    max-width: 80px;
+    border-top-left-radius: 3px;
+    border-top-right-radius: 0;
```

[design polish] The bottom left corner is missing the 3px radius in the largest view.

```suggestion
    border-top-right-radius: 0;
    border-bottom-left-radius: 3px;
```